docs/application/dotnet/guides/nui/custom-layout.md

```diff
@@ -0,0 +1,165 @@
+# Create Custom Layout
```

Comment:
Global comment:

Kindly note to update the headings with Sentence Case and Note style as suggested in the styleguide:

- Headings: https://github.com/Samsung/tizen-docs#note
- Note style: https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md

Reply:
> 
> 
> Global comment:
> 
> Kindly note to update the headings with Sentence Case and Note style as suggested in the styleguide:
> 
>     * Headings: https://github.com/Samsung/tizen-docs#note
> 
>     * Note style: https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md

@rishu-samsung
I told Surbhi that you guys update heading style not to leave comment like this. There are too many comments on this PR. It is hard to open this PR.